packages/block-editor/src/components/block-bindings-toolbar-indicator/style.scss

```diff
@@ -0,0 +1,10 @@
+.block-toolbar-block-bindings-indicator {
+	display: inline-flex;
+	align-items: center;
+	height: 48px;
+	padding: 6px;
+}
+
+.block-editor-block-bindings-indicator svg g {
+	stroke: #9747ff;
```

Comment:
@jasmussen or @SaxonF - do you usually put colors that are used in several places under a variable name?

Reply:
1. Should we reuse this property ?https://github.com/WordPress/gutenberg/blob/3f9fa3bc2622ef8af610816e3c75e1a387c159c0/packages/base-styles/_default-custom-properties.scss#L7

2. Currently the color of the "bound" blocks is `#9747ff` which is close but different from synced blocks color (`#7a00df`). Should it be the same?

Reply:
OK, in 4dc11032bead87178814029fd0593c0a66ce6cf0, I've added a new `--wp---wp-bound-block-color` property to [_default-custom-properties.scss](https://github.com/WordPress/gutenberg/blob/3f9fa3bc2622ef8af610816e3c75e1a387c159c0/packages/base-styles/_default-custom-properties.scss#L7) that I reused in other components.

We can still change this color in the future.